tools/sense_studio/templates/testing.html

```diff
@@ -0,0 +1,136 @@
+            <div>
+                <label class="uk-form-label" for="title">Window Title</label>
+                <div class="uk-form-controls">
+                    <div class="uk-inline uk-width-1-1">
+                        <span class="uk-form-icon" uk-icon="icon: pencil"></span>
+                        <input class="uk-input" type="text" id="title" placeholder="Title" autocomplete="off" />
+                    </div>
+                </div>
+            </div>
+            <div class="uk-margin-top">
+                <label class="uk-form-label">Choose Input Source</label>
+                 <div class="uk-form-controls">
+                     <div class="uk-inline uk-width-1-1">
+                        <label>
+                            <input class="uk-radio" type="radio" name="inputSource" value="0" onclick="toggleInputVideoField();" checked>
+                            &nbsp;&nbsp;Use Webcam
+                        </label>
+                        <label class="uk-margin-large-left">
+                            <input class="uk-radio" type="radio" name="inputSource" value="1" onclick="toggleInputVideoField();">
+                            &nbsp;&nbsp;Input Video
+                        </label>
+                     </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="uk-child-width-1-1@s uk-grid-small" uk-grid>
+            <div>
+                 <div class="uk-form-controls">
+                     <div class="uk-width-1-1">
+                        <label>
+                            <input class="uk-checkbox" type="checkbox" name="saveVideo" onclick="toggleOutputVideoField();">
+                            &nbsp;&nbsp;Save Output Video
+                        </label>
+                     </div>
+                </div>
+            </div>
+        </div>
+
+        <div id="inputVideoDiv" class="uk-child-width-1-1@s uk-grid-small uk-hidden" uk-grid>
+            <div>
+                <label class="uk-form-label" for="inputVideoPath">Input video path</label>
+                <label class="uk-form-label uk-text-danger" id="inputVideoPathLabel"></label>
+                <div class="uk-form-controls">
+                    <div class="uk-inline uk-width-1-1">
+                        <span class="uk-form-icon" uk-icon="icon: video-camera"></span>
+                        <input class="uk-input file-path-search" type="text" id="inputVideoPath" placeholder="/path/to/input-video"
+                               autocomplete="off" oninput="getPathInputs('inputVideoPathLabel', 'inputVideoPath');">
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div id="outputVideoDiv" class="uk-child-width-1-1@s uk-grid-small uk-hidden" uk-grid>
+             <div>
+                <label class="uk-form-label" for="outputVideoName">
+                    Output video name ({{ output_path_prefix }}..) <span class="uk-label">.mp4</span>
```

Comment:
The order of fields is weird here, because if both optional fields are shown it becomes
1. Save video
2. Input video path
3. Output video path